packages/docusaurus-theme-classic/src/theme/NavbarItem/DefaultNavbarItem.tsx

```diff
@@ -83,6 +83,12 @@ function DefaultNavbarItemDesktop({
       {...props}
     />
   );
+
+  if (isDropdownItem) {
+    element = cloneElement(<li />, {}, element);
```

Comment:
why use `cloneElement` here instead of `<li>{element}</li>`

cloneElement is quite often an antipattern to avoid

Reply:
Yes, you're right, I just thought that `key` prop should be forwarded to parent `<li>` element, but it doesn't seem to matter in this case.